schematic/manifest/generator.py

```diff
@@ -744,7 +744,11 @@ def get_dataframe_by_url(self, manifest_url: str) -> pd.DataFrame:
         wb = sh[0]
 
         # get column headers and read it into a dataframe
-        manifest_df = wb.get_as_df(hasHeader=True).drop(columns="")
+        manifest_df = wb.get_as_df(hasHeader=True)
+
+        # An empty column is sometimes included
+        if "" in manifest_df:
+            manifest_df.drop(columns=[""], inplace=True)
```

Comment:
Looks good! You're dropping the empty column only if it's present in the dataframe.